### DISCUSSION TITLE
Removed and Re-Added my academic mail and Lost my github student developer pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#69974] [GitHub Education]
### Hi everyone,

I just accidently removed my academic email from my github account. Then i realise that my student developer pack is gone. I have re-added the email into my account, but still I didn't get back my student developer pack. Also I can not re-apply a new application, it shows that I should contact in support. What should I do now?

**Comment by mohit-nagaraj** [comment]
Sup. Losing access to your GitHub Student Developer Pack can be concerning, but there are steps you can take to resolve this issue:

1. **Contact GitHub Support**: Since you've mentioned that you can't re-apply and are instructed to contact support, that's the best course of action. Go to GitHub's support page and submit a request explaining your situation. Provide all the necessary details, such as your academic email, GitHub username, and any additional information they may require.

2. **Verify Your Student Status**: Ensure that your academic email is correctly associated with your student status. Sometimes it can take a little time for the Student Developer Pack to be reinstated after email changes. If you're still a student, this should be a straightforward process with GitHub Support's assistance.

3. **Patience**: While waiting for a response from GitHub Support, be patient. They'll work to help you resolve the issue, but it may take some time.

GitHub Support is your best resource for dealing with account-related issues, and they should be able to assist you in reactivating your Student Developer Pack. Remember to provide them with all necessary information to expedite the process.

I hope this helps, and best of luck in getting your Student Developer Pack back! 😊 If you found this information helpful, feel free to accept the answer.
